website/server/controllers/api-v3/user.js

```diff
@@ -28,7 +28,29 @@ let api = {};
  * @apiGroup User
  *
  * @apiSuccess {Object} data The user object
- */
+ *
+ * @apiDescription The user profile contains data related to the authenticated user including (but not limited to);
+ * Achievements
+ * Authentications (including types and timestamps)
```

Comment:
It's never too late...  I put the description back in, and I moved the link to the model into an example